src/inference/include/openvino/runtime/intel_myriad/myriad_properties.hpp

```diff
@@ -0,0 +1,133 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <openvino/runtime/properties.hpp>
+#include <string>
+
+namespace ov {
+namespace intel_myriad {
+namespace common {
```

Comment:
do we need a special namespace `common`? Maybe we can keep all properties in the `intel_myriad` directly..

Reply:
Changed